packages/scrum-server/src/api/components/user/service.js

```diff
@@ -45,9 +45,10 @@ const signUp = async req => {
   }
 };
 
+// TODO remove the req 
 const localAuth = async req => {
   const token = req.cookies.auth;
-
+  console.log('token', token);
```

Comment:
is this console expected?

Reply:
Kind of, ill remove it for now, we need better ways to troubleshoot errors